modules/layers/src/column-layer/column-layer-vertex.glsl.js

```diff
@@ -74,10 +78,14 @@ void main(void) {
   vec3 normals_commonspace = project_normal(vec3(rotationMatrix * normals.xy, normals.z));
 
   if (extruded) {
-    vec3 lightColor = lighting_getLightColor(instanceColors.rgb, project_uCameraPosition, position_commonspace.xyz, normals_commonspace);
-    vColor = vec4(lightColor, instanceColors.a * opacity) / 255.0;
+    if (isWireframe) {
```

Comment:
Use `if (extruded && !isWireFrame)` to save a switch case.

Reply:
@xintongxia Lighting should be disabled on the wireframe